### PR TITLE
chore: Add default coreapi-release to keep docker build happy

### PR DESCRIPTION
**Pull request by arajkumar**
This fixes a problem when running `docker build`. It is not an issue in CI/CD as it uses makefiles which handles this case., it is just to make developer life happy.

### CODE REVIEW

File: hack/coreapi-release-template [hunk 1]
```diff
@@ -0,0 +1,3 @@
+COMMITTED_AT=1970-01-01 00:00:00 +0000
+COMMIT_HASH=badbadf
```
Comment (deepak1725) [inline hack/coreapi-release-template:2]:
```suggestion
COMMIT_HASH=randomhash
```

**Review by deepak1725 — CHANGES_REQUESTED** [review]

File: hack/coreapi-release-template (hunk 1 above)
Comment (arajkumar) [reply hack/coreapi-release-template:2]: Your suggestion is not a valid time stamp and hash.

Comment (arajkumar) [reply hack/coreapi-release-template:2]: It is verified from smoke test

Comment (deepak1725) [reply hack/coreapi-release-template:2]: Isn't it a placeholder..?

Comment (arajkumar) [reply hack/coreapi-release-template]: when running smoke test on images build using `docker build` should pass right?
e.g. 
1. you built image using `docker build . -t foo` and deployed on your dev cluster
2. executing smoke test on the dev cluster should pass

**Review by deepak1725 — APPROVED** [review]

**Review by sawood14012 — APPROVED** [review]
LGTM

**Pull request merged**